app/controllers/api/v0/analytics_controller.rb

```diff
@@ -57,9 +57,12 @@ def load_owner
       end
 
       def validate_date_params
-        raise ArgumentError, "Required 'start' parameter is missing" if analytics_params[:start].blank?
+        if analytics_params[:start].blank?
+          raise ArgumentError,
+                I18n.t("api.v0.analytics_controller.start_missing")
+        end
```

Comment:
I think it's a bit strange to have two different ways to essentially do the same thing right after each other. Maybe follow the same pattern as below:

```suggestion
       message = I18n.t("api.v0.analytics_controller.start_missing")
       raise ArgumentError, message if analytics_params[:start].blank?
```

Reply:
Yes, I don't remember it, maybe the linter did it at some timing.

I do think this just works even with indent:
``` ruby
        raise ArgumentError, I18n.t("api.v0.analytics_controller.start_missing") if analytics_params[:start].blank?
```

Reply:
Yes, and maybe the linter complicated it at some timing.

Does it just works like this?
``` ruby
        raise ArgumentError, I18n.t("api.v0.analytics_controller.start_missing") if analytics_params[:start].blank?
        raise ArgumentError, I18n.t("api.v0.analytics_controller.date_format") unless valid_date_params?
```

Reply:
It would but the lines seem pretty long. As long as they stay within our limit I'd be ok with that though.